solidity/contracts/Greeter.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity >=0.8.4 <0.9.0;
+pragma solidity >=0.8.18 <0.9.0;
```

Comment:
is it using anything that disallows it to be compiled with sol 0.8.4?

Reply:
@wei3erHase the template does not have many features and can be compiled with any version, really. But we tend to upgrade to a more recent version after forking the template, and in addition to that, the change would promote the use of a newer compiler. 0.8.4 was 2+ years ago 👴🏻